HIP/hip-0000-event-serialization.md

```diff
@@ -0,0 +1,227 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+working-group: Swirlds Labs Platform Hashgraph Team
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the round during which an event was created to the event's meta-data.
```

Comment:
I believe the generally accepted spelling is "metadata", as opposed to "meta-data" https://en.wikipedia.org/wiki/Metadata

Reply:
indeed it is.  :-)